raft/doc.go

```diff
@@ -180,5 +180,100 @@ cannot be removed any more since the cluster cannot make progress.
 For this reason it is highly recommended to use three or more nodes in
 every cluster.
 
+MessageType
+
+Package raft sends and receives message in Protocol Buffer format (defined
```

Comment:
Package raft sends and receives messages in the Protocol Buffer format defined in the raftpb package.